tests/e2e/standard/forms/immunization-visit.standard-wdio-spec.js

```diff
@@ -0,0 +1,170 @@
+const utils = require('../../../utils');
+const loginPage = require('../../../page-objects/login/login.wdio.page');
+const commonPage = require('../../../page-objects/common/common.wdio.page');
+const contactPage = require('../../../page-objects/standard/contacts/contacts.wdio.page');
+const reportsPage = require('../../../page-objects/reports/reports.wdio.page');
+const analyticsPage = require('../../../page-objects/analytics/analytics.wdio.page');
+const genericForm = require('../../../page-objects/forms/generic-form.wdio.page');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const userFactory = require('../../../factories/cht/users/users');
+const gatewayApiUtils = require('../../../gateway-api.utils');
+const immVisitForm = require('../../../page-objects/forms/immunization-visit.wdio.page');
```

Comment:
I thought about that, the thing is that this immunization page will not be used only for `standard` because there is a test in `brac` for the immunization and I think that the page can be used also in that test.
I think that we will need to find a different way to organize the `page-object` files.